front/app/containers/MainHeader/Components/NavbarContent/MobileNavbarContent/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { Suspense, useRef, useState } from 'react';
+
```

Comment:
Most of this code was just copied into this new component to separate the desktop + mobile nav bars. Only made some small styling tweaks + added the hamburger menu.